helix-core/src/test/java/org/apache/helix/integration/multizk/TestMultiZkHelixJavaApis.java

```diff
@@ -86,7 +86,7 @@ public class TestMultiZkHelixJavaApis extends TestMultiZkConnectionConfig {
 
   @BeforeClass
   public void beforeClass() throws Exception {
-    super.beforeClass();
+    super.beforeApiClass();
```

Comment:
If my understanding is correct then you are bypassing _zkClient object creation from TestMultiZkConnectionConfig class. If that's true then we should add few null checks in TestMultiZkConnectionConfig.setupCluster or other methods. This is just to make sure that any user of TestMultiZkConnectionConfig now expects to initialize _zkClient object.

Reply:
Another point, i think we are not even using _zkClient anywhere in the test here right? if not we can completely remove it from this test.

Reply:
we are using indirectly through base class. so i can't remove.